server/server.js

```diff
@@ -38,6 +38,16 @@ function prodLangs() {
 
 const availableLanguages = conf.l10n_dev ? allLangs() : prodLangs();
 
+const envURL = (env) => {
```

Comment:
using this to set the URL so this can be tested on dev environments. Not super familiar with convict, but maybe this would be better done in config